### PR TITLE
Update 'Logging Off' (Celine's piece) with final text + minor styling changes

### PR DESCRIPTION
**Pull request by celinenguyen**
(@maisutton it doesn't look like I can add you as a formal reviewer, but wanted you to see this!)

This updates my piece with some copy changes Mai and I agreed to on the [🔒 Google Doc](https://docs.google.com/document/d/1joOA6DiKNSTQaupqSDCxplOTBgcN1QaT_9TiJXwBUNY/edit?usp=sharing) (only shared with COMPOST people). Mai, a few notes:

* Per my comments on the Google Doc, I rearranged your suggested wording clarification on `016a` but kept the spirit of the change, which does clarify Daniella's intent
* I added a new section to the resources page under '**Foraging for fruit**' because I wanted to add the [Falling Fruit](https://fallingfruit.org/) link for anyone interested!
* Per your suggestion to add Tal's suggested links (I don't think I can tag Tal here?) I added a link to the Native Plant Finder and Native Land under the heading '**The bioregionalism quiz**'

I also cherry-picked a styling change to the season icon indicators (so they're coloured!) that I forgot to push up last week…sorry about that! (Mentioned this briefly in the Elements chat)

### CODE REVIEW

**Comment by celinenguyen** [comment]
Sorry, just pushed another commit because the `/resources` directory wasn't pushing up (renamed it to `/notes` and it seemed to work fine). Not sure if 'resources' is reserved for special Hugo behavior? Anyways, everything should work now!

File: content/pieces/logging-off/016a/index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ You are so relieved that Daniella’s text comes as a surprise: *I can’t stand
 
 You watch her texts come in, line after line, filling up your screen.
 
-*It’s just very obvious that the whole thing seemed irrelevant to people. Not worth talking about. It’s just this inexplicable, historical accident that we live here and others have been pushed to the margins. This ambiguous, distant tragedy that’s completely irrelevant to our lives today, strolling through the forest and seeing how small and impoverished it becomes every year.*
+*It’s very obvious that the whole thing seemed irrelevant to people. Not worth talking about. They act like it's this inexplicable, historical accident that we live here and others have been pushed to the margins. An ambiguous, distant tragedy that’s completely irrelevant to our lives today, strolling through the forest and seeing how small and impoverished it becomes every year.*
```
Comment (celinenguyen) [inline content/pieces/logging-off/016a/index.md:38]: @maisutton here's where I made a slight change to your suggested clarification

File: content/pieces/logging-off/notes/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+layout: multipage
+order: celine
+# custom frontmatter
+---
+
+### Foraging for mushrooms
+
+If you are interested in mushroom foraging, there are a number of regional mycological societies that do mushroom walks. It’s very likely that the protagonist of this story would have consulted information from organizations like the [Bay Area Mycological Society](http://bayareamushrooms.org/) and [Mycological Society of San Francisco](https://www.mssf.org/).
+
+### Foraging for fruit
```
Comment (celinenguyen) [inline content/pieces/logging-off/notes/index.md:13]: @maisutton this part wasn't in the Google Doc (realised I forgot to link the Falling Fruit site!) so wanted to flag it for you

File: content/pieces/logging-off/notes/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Urban foraging can yield a surprising amount of fruit, edible plants, and even mushrooms. As Daniella notes, you are legally allowed to take fruit that hangs over a public sidewalk in California. (She may have read [this *Berkeleyside* article about it](https://www.berkeleyside.org/2016/06/15/a-guide-to-urban-fruit-foraging-in-the-east-bay).) If you're interested in urban fruit foraging, [Falling Fruit](http://fallingfruit.org/) is an international map to help you find trees in your area.
+
+### The bioregionalism quiz
+
+I first came across the [bioregionalism quiz](https://dces.wisc.edu/wp-content/uploads/sites/128/2013/08/Where-You-At-Quiz.pdf) that Alice mentions in the novel [*Weather* by Jenny Offill](https://bookshop.org/books/weather-9780345806901/9780385351102).
+
+If you live in the U.S. and want to know the native plants in your area, try the [Native Plant Finder](https://www.nwf.org/nativeplantfinder/plants).
```
Comment (celinenguyen) [inline content/pieces/logging-off/notes/index.md:21]: @maisutton lines 21 and 23 are where I've added the resources Tal suggested

**Comment by uditvira** [comment]
Looks good to me!

**Review by uditvira — APPROVED** [review]

**Pull request merged**